addons/l10n_ar/data/uom_uom_data.xml

```diff
@@ -13,48 +13,18 @@
     <record model='uom.uom' id='uom.product_uom_unit'>
         <field name='l10n_ar_afip_code'>07</field>
     </record>
-    <record model='uom.uom' id='uom.product_uom_dozen'>
-        <field name='l10n_ar_afip_code'>09</field>
-    </record>
-    <record model='uom.uom' id='uom.product_uom_cm'>
-        <field name='l10n_ar_afip_code'>20</field>
-    </record>
     <record model='uom.uom' id='uom.product_uom_day'>
         <field name='l10n_ar_afip_code'>98</field>
     </record>
-    <record model='uom.uom' id='uom.product_uom_floz'>
-        <field name='l10n_ar_afip_code'>98</field>
-    </record>
-    <record model='uom.uom' id='uom.product_uom_foot'>
```

Comment:
Indeed, we decided yesterday later afternoon that we will archive all of the UoMs that were marked for deletion (in the spec) instead of deleting them and will undo the changes to the localization tests (except maybe un-archiving the records if needed for them to run properly)